internal/infrastructure/google/google.go

```diff
@@ -0,0 +1,17 @@
+package google
+
+import (
+	"github.com/reearth/reearth-backend/internal/usecase/gateway"
+	"github.com/reearth/reearth-backend/pkg/file"
+)
+
+type google struct {
+}
+
+func NewGoogle() gateway.Google {
+	return &google{}
+}
+
+func (g google) FetchCsv(token string, fileId string, sheetName string) (*file.File, error) {
```

Comment:
```suggestion
func (g google) FetchCSV(token string, fileId string, sheetName string) (*file.File, error) {
```

In golang, all abbreviations may be capitalized.

Reply:
Done